### DISCUSSION TITLE
Plant thumbnail/avatar picture, is uploaded from server every time

### DISCUSSION THREAD

**Discussion by dobrKot** [MDeLuise/plant-it#362] [Q&A]
Hey, good morning. Not sure if it's an issue, expected behaviour or just mis-configuration. The problem is that I have added 4 plants, and added 1 image to each of them. Each of the images is between 2-3mb. But when I scroll between this 4 plants, client is requesting original image each time(it's visible in browser console) instead of caching, which is insane and turn all UI into slide-show. Can I overcome it somehow or it's an expected behaviour?

**Comment by dobrKot** [comment]
Could it be connected with this response?

accept-ranges:
bytes

access-control-allow-origin:
*
cache-control:
**no-cache, no-store, max-age=0, must-revalidate**
connection:
keep-alive
content-length:
3623159
content-type:
image/jpeg
date:
Fri, 15 Nov 2024 11:15:15 GMT
expires:
0
keep-alive:
timeout=60
pragma:
no-cache
vary:
Origin
vary:
Access-Control-Request-Method
vary:
Access-Control-Request-Headers
x-content-type-options:
nosniff
x-rate-limit-remaining:
997
x-xss-protection:
0

**Comment by fskaeh** [comment]
Same experience, looks like assets don't get cached in the client and something as simple as scrolling the plant carousel on the home page is causing pictures to get re-fetched from the server, making the UI lag a lot.
